src/commands/infra/scaffold.ts

```diff
@@ -38,8 +38,19 @@ export const scaffoldCommandDecorator = (command: commander.Command): void => {
             "You must specify each of the variables 'name', 'source', 'version', 'template' in order to scaffold out a deployment."
           );
         }
-        await copyTfTemplate(opts.template, opts.name);
-        await validateVariablesTf(path.join(opts.template, "variables.tf"));
+        const scaffoldDefinition = [opts.source, opts.template, opts.version];
```

Comment:
Is versions always required? Can I omit and have it simply pull master.

Reply:
No, version is a mandatory argument. For now, if the user wants to use master, they should use `--version master`.